traitlets/traitlets.py

```diff
@@ -864,9 +864,12 @@ def _notify_trait(self, name, old_value, new_value):
 
```

Comment:
Is this line note in the wrong spot? I think I've fixed what you're referring to, but it wasn't here.